### PR TITLE
CassandraLogger prefix made configurable

### PR DESCRIPTION
**Pull request by vlezhnin**
Prefix is optional and configurable by system settings. By default value is `reaktive`
@jypma @alar17

### CODE REVIEW

File: ts-reaktive-actors/src/main/java/com/tradeshift/reaktive/cassandra/CassandraLogger.java [hunk 1]
```diff
@@ -25,9 +25,12 @@ public static void apply(ActorSystem system) {
         String statsdhost = system.settings().config().getString("codahale.statsd.hostname");
         int statsdport = system.settings().config().getInt("codahale.statsd.port");
         long interval = system.settings().config().getDuration("codahale.statsd.tick-interval", TimeUnit.MILLISECONDS);
-        
+        String prefix = "reaktive";
```
Comment (jypma) [inline ts-reaktive-actors/src/main/java/com/tradeshift/reaktive/cassandra/CassandraLogger.java]: don't put defaults in code. Instead, have a `reference.conf` resource in the root of this module's classpath with the default. Then your code can just say `.getString`.

And wouldn't a more sensible default be `"cassandra-client"`?

Comment (vlezhnin) [inline ts-reaktive-actors/src/main/java/com/tradeshift/reaktive/cassandra/CassandraLogger.java]: feedback addressed

File: ts-reaktive-actors/src/main/java/com/tradeshift/reaktive/cassandra/CassandraLogger.java [hunk 2]
```diff
@@ -25,9 +25,9 @@ public static void apply(ActorSystem system) {
         String statsdhost = system.settings().config().getString("codahale.statsd.hostname");
```
Comment (jypma) [inline ts-reaktive-actors/src/main/java/com/tradeshift/reaktive/cassandra/CassandraLogger.java]: actually, where's the rest of the defaults? Looks like they never made it in either. Could you put some sensible defaults in for the rest? Then it's all :+1: 

You probably want to use the `codahale.statsd { ... }` syntax for that, see https://github.com/typesafehub/config#examples-of-hocon .

**Comment by jypma** [comment]
LGMT. Ok to merge?

**Pull request merged**